packages/core/src/amazonq/webview/ui/tabs/constants.ts

```diff
@@ -24,11 +24,11 @@ export const TabTypeDataMap: Record<TabType, TabTypeData> = {
     featuredev: {
         title: 'Q - Dev',
         placeholder: 'Describe your task or issue in as much detail as possible',
-        welcome: `Hi, I'm the Amazon Q Developer Agent for software development.
+        welcome: `Hi! I'm the Amazon Q Developer Agent for software development. 
+        
```

Comment:
Please make sure this welcome message is formatted correctly when you load up the extension. I've had to fix those issues so many times 😅

Reply:
<img width="343" alt="image" src="https://github.com/user-attachments/assets/466e7d2d-a239-4f24-ac1a-bd39aff7831e">